examples/gallery/index.html

```diff
@@ -36,11 +43,38 @@ <h1>Slint Gallery</h1>
     </a>
   </p>
   <script type="module">
-    import init from './pkg/gallery.js';
-    init().finally(() => {
-      document.getElementById("spinner").remove();
-    });
+    var fluent = `./pkg/fluent/gallery.js`;
+    var material = "./pkg/material/gallery.js";
```

Comment:
Just a small nitpick :)

```suggestion
    const fluent = "./pkg/fluent/gallery.js";
    const material = "./pkg/material/gallery.js";
```

Maybe the variables/constants should be called "fluentImport" or "fluentImportPath" to make it clearer that what this is.